lib/sinatra/base.rb

```diff
@@ -1743,29 +1743,21 @@ def cleaned_caller(keep = 3)
       end
     end
 
-    # Fixes encoding issues by
-    # * defaulting to UTF-8
-    # * casting params to Encoding.default_external
-    #
-    # The latter might not be necessary if Rack handles it one day.
-    # Keep an eye on Rack's LH #100.
-    def force_encoding(*args) settings.force_encoding(*args) end
-    if defined? Encoding
-      def self.force_encoding(data, encoding = default_encoding)
-        return if data == settings || data.is_a?(Tempfile)
-        if data.respond_to? :force_encoding
-          data.force_encoding(encoding).encode!
-        elsif data.respond_to? :each_value
-          data.each_value { |v| force_encoding(v, encoding) }
-        elsif data.respond_to? :each
-          data.each { |v| force_encoding(v, encoding) }
-        end
-        data
+    # Fixes encoding issues by defaulting to UTF-8
```

Comment:
Is this correct? According to [the original commit](https://github.com/sinatra/sinatra/pull/56/commits/08a62a2da3a84814cd2e89f60542302886c49772), it explicitly set `UTF-8` to default. But now, it's removed.

Reply:
It looks like it was added back:

https://github.com/sinatra/sinatra/blob/7b7043cde3515afde713b4da3e54443ca831d3df/lib/sinatra/base.rb#L1781

Reply:
Well why did you still put the comment in the same place?

Reply:
Where would you like it?

Reply:
Where placed above this line, it should match the content of the comment.
https://github.com/sinatra/sinatra/blob/7b7043cde3515afde713b4da3e54443ca831d3df/lib/sinatra/base.rb#L1781

Also, I'm not sure if it is correct about removing "casting params to Encoding.default_external".
We have still been forcing encoding for params.

Reply:
That was an old comment when params were cast to Encoding.default_external with no option to override. I don't think that would be the appropriate location for that comment either.

Reply:
I updated the comment to be more relevant to the method. Does that look better to you?

Reply:
OK great